### PR TITLE
Long filename fix 🗄

### PR DESCRIPTION
**Pull request by The-Robin-Hood**
**PR** : The pull request addressed an issue with long file names in the UI 
**Issue** : #76

#### Before: 
image

#### After:
image

### CODE REVIEW

**Review by dephraiim — COMMENTED** [review]
The Dialog component is currently in the process of being reusable. Modifying the title in the component could limit the component's reusability.

The issue also affects the signed page

**Comment by The-Robin-Hood** [comment]
How about we having extra param - boolean in Dialog Component and to shorten the title if that param is only true ?

**Comment by Mythie** [comment]
Yep I'm for a `truncateTitle` prop so we can opt in and out as desired. Obviously make it opt-out by default 😄

**Comment by The-Robin-Hood** [comment]
@dephraiim Hope this works. 
@Mythie Thanks for the suggestion

**Review by ElTimuro — DISMISSED** [review]
- Checked and works
- The signed page still needs to be fixed (and all other places, where those properties are used)
- @The-Robin-Hood Do you want to fix those too or should I create a new ticket?

**Comment by The-Robin-Hood** [comment]
I suggest creating a new ticket to address those specific issues. This will allow us to track and prioritize the necessary fixes separately from the current pull request. Thank you.

File: packages/ui/components/dialog/Dialog.tsx [hunk 1]
```diff
@@ -29,11 +31,14 @@ export function Dialog({
   formValues,
   setLoading,
   icon,
+  truncateTitle = true,
```
Comment (kulgavy) [inline packages/ui/components/dialog/Dialog.tsx:34]: Is it used? 🤔

Comment (Mythie) [reply packages/ui/components/dialog/Dialog.tsx:34]: Good catch! Updated to fix that oversight on my end 😅

**Review by Mythie — APPROVED** [review]

**Pull request merged**